types/theo/index.d.ts

```diff
@@ -93,10 +93,10 @@ export function registerFormat<T extends string = never>(
     name: Format | T,
     format: FormatResultFn | string
 ): void;
-export function registerTransform<T extends string = never>(
-    name: Transform | T,
-    valueTransforms: ValueTransform[] | T[]
```

Comment:
Out of curiosity, why not `V extends string = never` and `V[]`?

Reply:
@amcasey Thanks for pointing that out. I'd thought I'd run into an error when doing it that way but after checking again, it appears to work fine as one would expect. Updated ready to merge 👍